java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/access/TruffleAccess.java

```diff
@@ -457,10 +457,11 @@ private static CurrentPCInfo getCurrentPosition(JPDADebugger debugger, JPDAThrea
     private static HitBreakpointInfo[] getBreakpointInfos(ExecutionHaltedInfo haltedInfo, JPDAThread thread) {
         ObjectVariable[] breakpointsHit = haltedInfo.breakpointsHit;
         ObjectVariable[] breakpointConditionExceptions = haltedInfo.breakpointConditionExceptions;
-        int n = breakpointsHit.length;
+        int n = (breakpointsHit != null) ? breakpointsHit.length : 0;
```

Comment:
No need to put `(...)` around the condition. All condition operators have higher precedence than `?:`.

Reply:
I know. I'm just used to it to visually enclose the condition for better readability.